README.md

```diff
@@ -1,30 +1,131 @@
 [![Build Status](https://travis-ci.com/fac-14/race-to-zero.svg?branch=master)](https://travis-ci.com/fac-14/race-to-zero) [![codecov](https://codecov.io/gh/fac-14/race-to-zero/branch/master/graph/badge.svg)](https://codecov.io/gh/fac-14/race-to-zero)
 
-## Overview
+# Race to Zero
+
+![](https://media.giphy.com/media/ful6IbWkBsVhe/giphy.gif)
 
-### Description
 Race to Zero helps users start achieving a zero waste lifestyle. Through simple challenges and information, users can discover easy ways to begin reducing their waste and learn why, tracking their progress through a points and achievements based gaming system. 
 
-### Team Roles
-[Joe](https://github.com/thejoefriel) - Scrum Master
-[Emma](https://github.com/SleepySheepy172) - DevOps
-[Armand](https://github.com/Armand-Lluka) - Researcher
-[Simon](https://github.com/dupreesi) - QA
+### The Team
+[Joe](https://github.com/thejoefriel) - Scrum Master | 
+[Emma](https://github.com/SleepySheepy172) - DevOps | 
+[Armand](https://github.com/Armand-Lluka) - Researcher | 
+[Simon](https://github.com/dupreesi) - QA 
 
 ### Tech Stack
 
-| Core | DevTools | Testing |
+| Core | Testing | Other |
 | - | -------- | -------- |
-|Node|Codecov|nodemon
-|Express|eslint|supertest
-|Handlebars|bcryptjs|nyc
-|PostgreSQL|body-parser|tap-spec
-|HTML|env2|tape
-|CSS|express||
-|Sass|express-handlebars||
-||serve-favicon||
+|Node|jest|bcryptjs
+|Express|supertest|pg-promise
+|Handlebars|eslint|body-parser
+|PostgreSQL|nyc|serve-favicon
+|HTML|codecov|env2|
+|CSS|nodemon||
+|Sass|||
+
+
+## Getting Started
+How to get a copy of the project up and running on your local machine.
+
+*Please ensure you have this software **installed and running** on your local machine **before** you attempt to run this webapp.*
+> **Node** (via nvm recomended)
+> see: https://github.com/creationix/nvm
+
+> **PostgreSQL**
+> see: https://wiki.postgresql.org/wiki/Detailed_installation_guides
+
+### Setup
+
+#### 1. Clone the repo:
+
+```
+$ git clone https://github.com/fac-14/race-to-zero.git
+$ cd race-to-zero
+```
+#### 2. Install *Dependencies* from NPM
+
+```
+$ npm i
+```
+
+#### 3. Ensure you have the Required Environment Variables
+
+create a `.env` file in your `race-to-zero` directory.                                                                            
+add a line for your `DATABASE_URL` variable and one for `TEST_DB_URL`.
+```sh
+DATABASE_URL = postgres://simon:pw123@localhost:5432/race_zero
+TEST_DB_URL = postgres://simon:pw123@localhost:5432/test_race2
```

Comment:
Maybe we should add a line telling developers to create a `simon` super user

Reply:
Thanks for the suggestion, we will implement that :+1: